src/main/java/jalau/at18/katas/game2048/team1/Main.java

```diff
@@ -0,0 +1,259 @@
+                    if (matrix[row][column] != 0 && matrix[row + 1][column] == 0) {
+                        matrix[row + 1][column] = matrix[row][column];
+                        matrix[row][column] = 0;
+                    } else if (matrix[row][column] != 0 && matrix[row + 1][column] == matrix[row][column]) {
+                        matrix[row + 1][column] = 2 * (matrix[row][column]);
+                        matrix[row][column] = 0;
+                    }
+                }
+            }
+        }
+        return matrix;
+    }
+
+    public int[][] moveLeft(int[][] matrix) {
+        for (int moves = 0; moves < MAX_ROW_MOVES; moves++) {
+            for (int column = 1; column < matrix.length; column++) {
+                for (int row = 0; row < matrix.length; row++) {
+                    if (matrix[row][column] != 0 && matrix[row][column - 1] == 0) {
+                        matrix[row][column - 1] = matrix[row][column];
+                        matrix[row][column] = 0;
+                    } else if (matrix[row][column] != 0 && matrix[row][column - 1] == matrix[row][column]) {
+                        matrix[row][column - 1] = 2 * (matrix[row][column]);
+                        matrix[row][column] = 0;
+                    }
+                }
+            }
+        }
+        return matrix;
+    }
+
+    public int[][] moveRigth(int[][] matrix) {
+        for (int moves = 0; moves < MAX_ROW_MOVES; moves++) {
+            for (int column = 2; column >= 0; column--) {
+                for (int row = 0; row < matrix.length; row++) {
+                    if (matrix[row][column] != 0 && matrix[row][column + 1] == 0) {
+                        matrix[row][column + 1] = matrix[row][column];
+                        matrix[row][column] = 0;
+                    } else if (matrix[row][column] != 0 && matrix[row][column + 1] == matrix[row][column]) {
+                        matrix[row][column + 1] = 2 * (matrix[row][column]);
+                        matrix[row][column] = 0;
+                    }
+                }
+            }
+        }
+        return matrix;
+    }
+
+    public boolean gameOver(int[][] matrix) {
+        int countZeros = countZerosMatrix(matrix);
+        Boolean verifySideUp = verifySideRow(matrix);
+        Boolean verifySideLeft = verifySideColumn(matrix);
+        boolean isGameOver = countZeros == 0 && !(verifySideUp) && !(verifySideLeft);
+
+        if (isGameOver) {
+            System.out.println("Game Over");
+        }
+        return isGameOver;
+    }
+
+    public int countZerosMatrix(int[][] matrix) {
+        int countZeros = 0;
+        for (int row = 0; row < matrix.length; row++) {
+            for (int column = 0; column < matrix.length; column++) {
+                if (matrix[row][column] == 0) {
+                    countZeros++;
+                }
+            }
+        }
+        return countZeros;
+    }
+
+    public Boolean verifySideRow(int[][] matrix) {
```

Comment:
Same about methods returning boolean, this could be `canRowSlide(...)` or something similar

Reply:
We already change the name